contrib/forms/hpi/FormHPI.class.php

```diff
@@ -45,13 +39,17 @@ class FormHPI extends ORDataObject
     var $factors;
     var $signs;
 
+    // suppress phpstorm undefined variable warning
+    var $string = null;
```

Comment:
This is not really needed since the $string variable is only used within the toString function (ie. doesn't need to be stored). I think if it were = below instead of .=, then the warning would likely go away